### ISSUE TITLE
[Tracking Issue] Improve Parameter Documentation

### ISSUE THREAD

**Issue by danieldouglas92** [GeodynamicWorldBuilder/WorldBuilder#766]
My lab-mate @PrajaktaPMohite and I were working on a worldbuilder file that included faults for one of her models and when we looked at the manual in the section called `Parameter Documentation/Features` we noticed that this section is almost entirely empty (except for the `Plume` feature). 

Similar to the developer / API documentation issue I thought I would create an issue for this part of the manual so that there is a reminder that this needs to be completed as well.

- [ ] Parameters / Features
    - [ ] Oceanic Plate
    - [ ] Subducting Plate
    - [ ] Mantle Layer
    - [ ] Continental Plate
    - [ ] Faults

@MFraters Is the idea that each of these features will be expanded into another drop down list showcasing the various temperature and composition models that they support as well?

**Comment by MFraters** [comment]
Yes, that is a good point. Thanks for creating this tracking issue! You can also add the plume to the list and check if off for completeness :)

> @MFraters Is the idea that each of these features will be expanded into another drop down list showcasing the various temperature and composition models that they support as well?

Yes, the idea of this sections is that if you quickly want to know how a feature works and what parameteres there are, this would be easier to read then the actual full parameter documentation. Although I don't know what format would work best for this, so if you or @PrajaktaPMohite want to give it a shot, feel free to experiment!
